### PR TITLE
Removed getting default rates from the API

### PR DESCRIPTION
**Pull request by mingazhev**
Partially fixes
https://sentry.happytravel.com/organizations/happytravel/issues/1961/?environment=Production&project=18&query=is%3Aunresolved

### CODE REVIEW

**Review by oleg1std — APPROVED** [review]

File: HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs [hunk 1]
```diff
@@ -201,6 +203,10 @@ private static TimeSpan GetTimeSpanToNextMinute()
             if (_cache.TryGetValue(cacheKey, out decimal result, GetTimeSpanToNextMinute()))
                 return result;
             
+            // TODO: Fast tests hotfix. Remove after tests will be rewritten
```
Comment (kirillta) [inline HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs:206]: A ticket required

File: HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs [hunk 2]
```diff
@@ -156,12 +159,11 @@ private static TimeSpan GetTimeSpanToNextMinute()
...
             foreach (var ((source, target), rate) in rates)
             {
-                var defaultRate = await GetDefaultRate(source, target);
-
                 ratesToStore.Add(new CurrencyRate
                 {
-                    Rate = defaultRate ?? rate,
-                    RateCorrection = rate - defaultRate ?? 0,
+                    Rate = rate,
+                    // TODO: Recheck if this is needed
```
Comment (kirillta) [inline HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs:165]: Why don't you do it now?

File: HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs (hunk 1 above)
Comment (mingazhev) [reply HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs:204]: Fixed

File: HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs (hunk 2 above)
Comment (mingazhev) [reply HappyTravel.Tsutsujigasaki.Api/Services/RateService.cs]: Fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**